tensorflow/lite/kernels/test_util.h

```diff
@@ -252,6 +252,9 @@ class SingleOpModel {
   void BuildInterpreter(std::vector<std::vector<int>> input_shapes,
                         bool allow_fp32_relax_to_fp16 = false);
 
+  void BuildInterpreter(std::vector<std::vector<int>> input_shapes,
```

Comment:
What about just a single method, with "num_threads = -1" as a new arg?

Reply:
Actually, since "allow_fp32_relax" is used rarely, better to make that the last arg.

Reply:
@jdduke : "allow_fp32_relax"  is already the last arg.